splice_machine/src/test/java/com/splicemachine/derby/utils/SpliceAdmin_OperationsIT.java

```diff
@@ -239,11 +246,9 @@ public void testKillLongRunningQuery(boolean useSpark) throws Exception {
         Thread thread = new Thread(new Runnable() {
             @Override
             public void run() {
-                PreparedStatement ps = null;
-                try (TestConnection connection = methodWatcher.createConnection()) {
-                    ps = connection.prepareStatement(sql);
-                    ResultSet rs = ps.executeQuery();
-                    assertTrue(rs.next());
+                try (TestConnection connection = methodWatcher.createConnection();
+                     PreparedStatement ps = connection.prepareStatement(sql);
+                     ResultSet rs = ps.executeQuery()) {
```

Comment:
we were verifying previously that the result set contained at least one row, this is removed now, is this intentional?

Reply:
Yes, this query was never completing in the first place (it's a long running query on purpose) and it was running in a separate thread, so even if the assertion failed it wouldn't have failed the test.